modules/account-lib/test/resources/trx/trx.ts

```diff
@@ -0,0 +1,52 @@
+// There is a file called "trx" in resources directory but it should be
+// removed in the future and add that data to this file.
+
+// # Contract call data fields
+
+export const FEE_LIMIT = '10000';
+export const BLOCK_NUMBER = 51407;
+export const BLOCK_HASH = '0000000000badb0d89177fd84c5d9196021cc1085b9e689b3e9a6195cac8bcae';
+export const MINT_CONFIRM_DATA = '2bf90baa1273140c3e1b5756b242cc88cd7c4dd8a61bf85cb5c1dd5f50ba61e066b53a15';
+export const EXPIRATION = 60000;
+
+export const PARTICIPANTS = {
+  from: {
+    address: 'TGeT2sfMYcjx3ra2HhQUvMyBcVhjBc1Lbk',
+    pk: 'EB94159C7EBC2959720B9F0321849F792B60FCBC7BFA4A9115F7E7A72B9ACE6F',
+  },
+  custodian: {
+    address: 'TVHsEa7nqPebk8fU5yc9ctf8n5X7DZKxkb',
+    pk: '280DA606E22BB0F857753D74B2450D80BBE0B03E70F15873180A932D689CEDD8',
+    bitcoinAddress: '2MytEhVhDLyEzmfeSA7yJ46yf5GhuKZ6gce',
+  },
+  merchant: {
+    address: 'TSQQkC8189ARqHhrpNgPAV1XLH9ouAwdPD',
+    pk: 'C87C22691E6423F48CFA2D2C5F60EB946E0DA6F31AEA3AA414D01B7670AA1092',
+    bitcoinAddress: '2NDjgTKnakBynrwSoyFpSinMsx9PPfubd2e',
+  },
+  multisig: {
+    address: 'TQVxJBfK4JHfJN9sbw4fGSzCLrH6c6c1d5',
+    pk: '0930B5929C4CE938BE6074EAA8B701E9FFF244A09327B6E92A3E875BFA1E7C34',
+  },
+};
```

Comment:
are these real anywhere?

Reply:
These are real addresses on Shasta testnet

Reply:
please do not add these to our unit tests

Reply:
they are live values with live keys, someone could basically abuse this and we need to rotate them

Reply:
Oh, ok, I have changed them